data/sql/updates/pending_db_world/rev_1722736427173275700.sql

```diff
@@ -0,0 +1,2 @@
+--
+UPDATE `gameobject_template` SET `Data1` = 61993, `Data2` = 32929 WHERE `entry` = 194108;
```

Comment:
This change is incorrect. The values for this gameobject are correct in current latest master AC according to a 53788 sniff, so they cannot be changed.

Reply:
I see, what would be the best place to override these values?

Reply:
IMO they shouldn't be overridden.

Reply:
Currently one of those values is overridden in code, at GameObject::Update, and it is necessary for the summon ritual to work.
Data1 which gets mapped to spellId contains a value for a spell that does not seem to exist, 62330.

Reply:
> [...] contains a value for a spell that does not exist, 62330.

The spell it points to is [a serverside spell](https://wowgaming.altervista.org/aowow/?spell=62330). Do we know what it does? No, but that's what is used and it shouldn't be overridden. Maybe a more knowledgeable contributor has a better idea or disagrees with me, but that is my opinion.

Reply:
Ok, I agree not to change this value in the database, I still need to learn a lot about the project and codebase.

So, how can I fix this?

What I found while investigating the issue is that the server checks specifically for that spell id (`if (spellId == 62330)`) and replaces the value to cast `61993` instead. This is how it currently works, how the server handles that serverside spell, it casts something else. Whether is the right way or not, I am not sure. Is there a similar scenario that I could take a look at?

Reply:
You didn't pick a simple issue to try to resolve. 😛

It looks to me like the 62330 casts should be scripted out in some manner. According to a sniff, it appears that the characters assisting with the ritual (i.e. clicking the portal) are channeling the 698 spell alongside the warlock until it is satisfied with the player count clicking the portal.

As well, I believe [this line](https://github.com/azerothcore/azerothcore-wotlk/blob/master/src/server/game/Entities/GameObject/GameObject.cpp#L1921) replaces the spell used for the 'closet' itself to the wrong one. It appears that it should also be `23598`, like regular meeting stones.

Reply:
What I ended up doing is moving these values into code. I don't think we would be worse than before, compared to the current implementation. It is an small improvement, it behaves as expected.